kuma/users/tests/test_tasks.py

```diff
@@ -43,8 +44,12 @@ def test_welcome_mail_for_verified_email(self):
                  email='welcome@tester.com',
                  password='welcome',
                  save=True)
-        request = RequestFactory().get('/')
+        request = self.rf.get('/')
         request.locale = 'en-US'
+        session = self.client.session
+        session.save()
+        request.session = session
+        self.get_messages(request)
```

Comment:
Could this also test if the list of messages contain the correct type of message?